Source/Parser/JsonLDParser.swift

```diff
@@ -226,19 +219,18 @@ class JsonLDParser {
 
 		return operations.reduce(into: [String: Operation]()) { (acc, operation) in
 			acc[operation.id] = operation
-		}		
+		}
```

Comment:
While we don't have a defined SF, it may be easier to keep as it is for now. Can you revert this hunk?